flink-ai-flow/ai_flow/ai_graph/ai_graph.py

```diff
@@ -36,7 +38,7 @@ class AIGraph(Graph):
     def __init__(self) -> None:
         super().__init__()
         self.nodes: Dict[Text, AINode] = {}
-        self._context_extractor: ContextExtractor = BroadcastAllContextExtractor()
+        self._context_extractor: bytes = cloudpickle.dumps(BroadcastAllContextExtractor())
```

Comment:
Though we had a talk about this offline, I am not sure why we should store it as bytes in AIGraph.

Reply:
At first, I want to make it easier to serialize AIGraph in json_utils when building workflow. But I think it is not necessary to serialize the extractor in workflow.